clients/java/zts/src/main/java/com/yahoo/athenz/zts/ZTSClient.java

```diff
@@ -524,18 +522,11 @@ public void setEnablePrefetch(boolean state) {
     }
 
     /**
-     * Set new ZTS Client configuration property. This method calls
-     * internal javax.ws.rs.client.Client client's property method.
-     * If already set, the existing value of the property will be updated.
-     * Setting a null value into a property effectively removes the property
-     * from the property bag.
-     * @param name property name.
-     * @param value property value. null value removes the property with the given name.
+     * Set the DNSResolver to be used by the ZTS Client
+     * @param resolver user supplied dns resolver
      */
-    public void setProperty(String name, Object value) {
```

Comment:
This public method was removed from ZTSClient which was probably used by existing clients of the library. Once they will update to the new version their code will not compile.
If this function can be safely ignored we can consider marking it as Deprecated and have it do nothing. Otherwise we should consider updating the README file with the API changes in the new version.

Reply:
there are multiple changes in the client libraries since we're moving from jax-rs to apache directly and we will have a separate section in the changelog when 1.0 is released.